fbw-a32nx/src/wasm/systems/a320_systems/src/hydraulic/mod.rs

```diff
@@ -3075,7 +3070,7 @@ impl A320BlueElectricPumpController {
     }
 
     fn has_overheat_fault(&self) -> bool {
-        self.has_low_level_fault
+        self.has_overheat_fault
```

Comment:
I hope this is correct, clippy complained and it made sense to me.

Reply:
Looks good also from my side.